workbench-core/base/src/aws/helpers/dynamoDB/dynamoDBService.ts

```diff
@@ -145,6 +146,32 @@ export default class DynamoDBService {
     return getter;
   }
 
+  /**
+   * retrieves item from DynamoDB table.
+   *
+   * @param key - single object of key to get for single get item
+   * @param params - optional object of optional properties to generate a get item request
+   * @returns Promise\<Record\<string,JSONValue\>\>
+   *
+   * @example Use this method to retrieve an item from ddb by Id
+   * ```ts
+   * const item = await dynamoDBService.getItem({'pk': 'pk', 'sk': 'sk'}, {projection: 'valueIWant'});
+   * ```
+   */
+  public async getItem(
```

Comment:
Should this function be used instead of `.get().execute()` going forward?

Reply:
correct, only for single item get